### PR TITLE
python bindings test collector copied

### PR DESCRIPTION
**Pull request by karol-bisztyga**

### CODE REVIEW

File: crates/cairo-lang-python-bindings/src/lib.rs [hunk 1]
```diff
@@ -87,11 +105,126 @@ fn starknet_cairo_to_casm(input_path: &str) -> Result<String, anyhow::Error> {
...
+                        }
+                    }
+                }
+            }
+        }
+    }
+    tests
+}
+
+
```
Comment (karol-bisztyga) [inline crates/cairo-lang-python-bindings/src/lib.rs:220]: this should be moved to a separate file

File: crates/cairo-lang-python-bindings/src/lib.rs [hunk 2]
```diff
@@ -87,11 +105,126 @@ fn starknet_cairo_to_casm(input_path: &str) -> Result<String, anyhow::Error> {
     Ok(casm)
 }
 
+#[pyfunction]
+fn call_test_collector(path: &str) -> PyResult<String> {
+    let plugins: Vec<Arc<dyn SemanticPlugin>> = vec![
+        Arc::new(DerivePlugin {}),
+        Arc::new(PanicablePlugin {}),
+        Arc::new(ConfigPlugin { configs: HashSet::from(["test".to_string()]) }),
+        Arc::new(StarkNetPlugin {}), // should we use it? it is used conditionally depending on args.starknet
```
Comment (MaksymilianDemitraszek) [inline crates/cairo-lang-python-bindings/src/lib.rs:114]: We probably don't need it, tests do not include any starknet code

**Review by MaksymilianDemitraszek — APPROVED** [review]
I don't see any high level issues here, If it works I am ok with merging it. We probably do not need starknet plugin though, as you mentioned in the comment.

File: crates/cairo-lang-python-bindings/src/lib.rs (hunk 2 above)
Comment (karol-bisztyga) [reply crates/cairo-lang-python-bindings/src/lib.rs]: 👍  I removed it

**Pull request merged**